src/main/resources/static/openapi/PrisonApi.json

```diff
@@ -28940,12 +28940,14 @@
           "entryDate": {
             "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}$",
             "type": "string",
+            "format": "date-time",
```

Comment:
I tried to find out how to add this to the relevant `@Schema` annotations within the prison-api (e.g. `format = "date-time"`), but I've drawn a blank so far.

Reply:
I'm not comfortable changing our copy of their spec either, but at least if someone overwrites it then the build will fail. They'd then have to change the corresponding builder and mapper etc, so hopefully that will be enough to make them stop in their tracks! 😄